Specs/Mocks/GlobeSurfaceTile/layer.json

```diff
@@ -0,0 +1 @@
":16200,"endY":13274,"startX":16187,"startY":13225},{"endX":16186,"endY":13265,"startX":16175,"startY":13257},{"endX":16186,"endY":13256,"startX":16172,"startY":13249},{"endX":16186,"endY":13248,"startX":16177,"startY":13241},{"endX":16186,"endY":13238,"startX":16177,"startY":13233},{"endX":16186,"endY":13232,"startX":16173,"startY":13225},{"endX":16176,"endY":13233,"startX":16173,"startY":13233},{"endX":16171,"endY":13256,"startX":16169,"startY":13255},{"endX":16174,"endY":13257,"startX":16169,"startY":13257},{"endX":16172,"endY":13231,"startX":16158,"startY":13225},{"endX":16157,"endY":13230,"startX":16147,"startY":13225},{"endX":16296,"endY":13003,"startX":16282,"startY":12979},{"endX":16296,"endY":13051,"startX":16201,"startY":13044},{"endX":16296,"endY":13043,"startX":16275,"startY":13036},{"endX":16296,"endY":13035,"startX":16106,"startY":13028},{"endX":16280,"endY":13027,"startX":16242,"startY":13019},{"endX":16295,"endY":13018,"startX":16269,"startY":13011},{"endX":16295,"endY":13020,"startX":16281,"startY":13019},{"endX":16283,"endY":13010,"startX":16282,"startY":13004},{"endX":16281,"endY":13010,"startX":16269,"startY":12979},{"endX":16270,"endY":13043,"startX":16174,"startY":13037},{"endX":16268,"endY":13018,"startX":16242,"startY":12979},{"endX":16241,"endY":13068,"startX":16230,"startY":13053},{"endX":16229,"endY":13053,"startX":16229,"startY":13052},{"endX":16241,"endY":13027,"startX":16233,"startY":13020},{"endX":16241,"endY":13019,"startX":16215,"startY":12979},{"endX":16226,"endY":13061,"startX":16201,"startY":13052},{"endX":16257,"endY":13052,"startX":16227,"startY":13052},{"endX":16228,"endY":13053,"startX":16227,"startY":13053},{"endX":16229,"endY":13027,"startX":16218,"startY":13021},{"endX":16204,"endY":13027,"startX":16120,"startY":13019},{"endX":16214,"endY":13019,"startX":16205,"startY":13019},{"endX":16232,"endY":13020,"startX":16205,"startY":13020},{"endX":16214,"endY":13018,"startX":16147,"startY":12979},{"endX":16200,"endY":13068,"startX":16189,"startY":13061},{"endX":16200,"endY":13060,"startX":16174,"startY":13044},{"endX":16173,"endY":13117,"startX":16160,"startY":13037},{"endX":16159,"endY":13105,"startX":16146,"startY":13053},{"endX":16159,"endY":13052,"startX":16133,"startY":13037},{"endX":16145,"endY":13068,"startX":16133,"startY":13053},{"endX":16146,"endY":13018,"startX":16120,"startY":13012},{"endX":16146,"endY":13011,"startX":16134,"startY":12978},{"endX":16132,"endY":13059,"startX":16092,"startY":13037},{"endX":16133,"endY":13011,"startX":16120,"startY":12987},{"endX":16133,"endY":12986,"startX":16121,"startY":12979},{"endX":16119,"endY":13027,"startX":16106,"startY":13020},{"endX":16119,"endY":13016,"startX":16106,"startY":13003},{"endX":16119,"endY":13002,"startX":16107,"startY":12986},{"endX":16120,"endY":12985,"startX":16100,"startY":12979},{"endX":16120,"endY":12986,"startX":16120,"startY":12986},{"endX":16093,"endY":13060,"startX":16092,"startY":13060},{"endX":16274,"endY":13036,"startX":16092,"start
```

Comment:
Are these files copies of what is store on Cesium ion? If so, this goes against the terms and conditions of downloading or caching data from that service. We shouldn't include them here. Is there perhaps a very minimal file we can use for the sake of testing? See the other files in `Specs/Data` for some examples.

Also, data for specs should go in the `Specs/Data` directory.